tests/session/cmd/run/test_common.py

```diff
@@ -0,0 +1,24 @@
+from argparse import ArgumentParser, ArgumentTypeError, Namespace
+from typing import Optional
+
+import pytest
+
+from tox.session.cmd.run.common import SkipMissingInterpreterAction
+
+
+@pytest.mark.parametrize("values", ["config", None])
+def test_call_SkipMissingInterpreterAction(values: Optional[str]) -> None:
+    argument_parser = ArgumentParser(description="Test SkipMissingInterpreterAction.")
```

Comment:
Don't add any information in tests that are not needed; e.g. here the description.